cppcheck-studio.py

```
@@ -0,0 +1,259 @@
+        # Check if it's a valid CPPCheck format
+        if not isinstance(data, dict):
+            return False, "Invalid format: Expected JSON object", None
+            
+        if 'issues' not in data:
+            # Try wrapping array in issues key for compatibility
+            if isinstance(data, list):
+                data = {'issues': data}
+            else:
+                return False, "Invalid format: Missing 'issues' key", None
+                
+        issue_count = len(data.get('issues', []))
```

Ruff (0.12.2)

Local variable `issue_count` is assigned to but never used

Remove assignment to unused variable `issue_count`

(F841)